src/Stratis.Bitcoin.Features.Interop/EthereumClient/EthereumClientBase.cs

```diff
@@ -0,0 +1,135 @@
+﻿using System.Collections.Generic;
+using System.Numerics;
+using Nethereum.ABI;
+using Nethereum.Contracts;
+using Nethereum.Hex.HexConvertors.Extensions;
+using Nethereum.Hex.HexTypes;
+using Nethereum.JsonRpc.Client;
+using Nethereum.RPC.Eth.Blocks;
+using Nethereum.RPC.Eth.DTOs;
+using Nethereum.Web3;
+using Nethereum.Web3.Accounts.Managed;
+using Stratis.Bitcoin.Features.Interop.Models;
+
+namespace Stratis.Bitcoin.Features.Interop.EthereumClient
+{
+    public class EthereumClientBase : IEthereumClientBase
```

Comment:
Please add xmldoc for methods in the class (and also for other methods in this PR that are not documented)